controller/src/main/java/io/pravega/controller/store/stream/StreamMetadataStore.java

```diff
@@ -223,19 +272,21 @@
      * @param newRanges      new key ranges to be added to the stream which maps to a new segment per range in the stream
      * @param sealedSegments segments to be sealed
      * @param scaleTimestamp timestamp at which scale was requested
+     * @param runOnlyIfStarted run only if the scale operation has already been started.
```

Comment:
Please explain the need of this parameter. It is an important design choice that should be brought out clearly.